docs/introduction/basic_usage.rst

```diff
@@ -0,0 +1,20 @@
+###########
+Basic usage
+###########
+
+[describe the steps a user needs to do to get the system into a usable state for someone
```

Comment:
That's a TODO I guess? Maybe add a comment to that effect - that way someone else can pick it up if they get to it before you.